src/encoding.iced

```diff
@@ -36,7 +36,7 @@ exports.Encoding = class Encoding
     # Left shift away all bits that are always going to be 0
     shift = @extra_bits {encoded_len, decoded_len : block.length }
```

Comment:
Let's make sure to do a broader cleanup in these functions, since a lot of the logic isn't needed anymore if we're not shifting.